models/entry.go

```diff
@@ -91,3 +91,7 @@ func (e *Entry) AsPointInTime() int64 {
 func (e *Entry) IsValuesEmpty() bool {
 	return len(e.Values) == 0
 }
+
+func (e *Entry) IsCodesPresent() bool {
+	return e.Coded.Codes != nil && len(e.Coded.Codes) != 0
+}
```

Comment:
You also put this exact function on `ResultValue`, so it seems like it might make the most sense to put it as a function on `Coded`.